rummager/config/deploy.rb

```diff
@@ -49,10 +49,15 @@
     teardown_connections_to(find_servers)
   end
 
-  desc "Restart rummager's publishing-api listener"
+  desc "Restart rummager's publishing-api listener for link updates"
   task :restart_publishing_api_listener do
     run "sudo initctl start rummager-publishing-queue-listener-procfile-worker || sudo initctl restart rummager-publishing-queue-listener-procfile-worker"
   end
+
+  desc "Restart rummager's publishing-api listener for govuk-index"
+  task :insert_data_into_govuk do
```

Comment:
The task name is a bit inconsistent with the description and the link update task. It might be clearer to name it something like `restart_published_content_listener`.

I guess ideally we'd also rename the existing task to make it clear that it restarts the links listener, but I don't know if it has to match something in govuk-puppet.